content/applications/general/email_communication/faq.rst

```diff
@@ -1,174 +1,208 @@
-  a list of files containing the logs of the database. The log files are created everyday
-  at 5:00 AM UTC. The two last days are not compressed, while the older ones are, in order
-  to gain space. The naming of the files for Today and Yesterday are :file:`odoo.log` and
-  :file:`odoo.log.1`. For the following, they are named with their dates and compressed.
-  See the Odoo.sh documentation about :ref:`logs <odoosh/logs>`. Use the command ``grep`` and
-  ``zgrep`` (for the compressed ones) to search through the files.
+.. tip::
+  The folder :file:`~/logs/` (preferably accessed by the command line) of an Odoo.sh database
+  contains a list of files containing the logs of the database. The log files are created everyday
+  at 5:00 AM (UTC).
 
-- **Odoo Online** users won’t have access to their logs. However you can still contact
-  `Odoo Support <https://www.odoo.com/help>`_ , if you have a recurring issue
-  with the same client or domain.
+  The two last days are not compressed, while the older ones are, in order to gain space. The naming
+  of the files for Today and Yesterday are respectively: :file:`odoo.log` and :file:`odoo.log.1`.
+
+  For the following days, they are named with their dates, and then compressed. See the Odoo.sh
+  documentation about :ref:`logs <odoosh/logs>`. Use the command ``grep`` and ``zgrep`` (for the
+  compressed ones) to search through the files.
+
+- **Odoo Online** users won't have access to the logs. However `Odoo Support
+  <https://www.odoo.com/help>`_ can still be contacted, if there is a recurring issue with the same
+  client or domain.
 
 Get help from support
 ---------------------
 
 In order to get helped efficiently, please provide as much information as possible. Here is a list
-of what can be helpful:
+of what can be helpful when reaching out to support about an issue:
+
 
-- The **EML** of the email, stating for *Electronic Mail*, is the file format containing all the
-  technical information required for an investigation. The documentation of your own email provider
-  might help you on how to get your EML files. Once you get the EML of the email, adding it
-  in the attachment of your ticket is the most efficient way for us to investigate. The support
-  will mainly focus on redundant issues.
+The **EML** (or **headers**) of the email is the file format containing all the technical
+information required for an investigation. The documentation from the email provider might provide
+the process on how to get EML/header files. Once the headers of the email are obtained, adding it
+into the Odoo support ticket is the most efficient way for the Odoo Support team to investigate.
 
   .. seealso::
      - `Gmail documentation
        <https://support.google.com/mail/answer/29436>`_
      - `Outlook documentation
        <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
```

Comment:
this seealso is indented for no reason, scoot it back please 🙂 I don't think there is anything we can do about the URL being over 100 chars, @StraubCreative any ideas?

```suggestion
.. seealso::
   - `Gmail documentation
     <https://support.google.com/mail/answer/29436>`_
   - `Outlook documentation
     <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
```